### PR TITLE
Consistent {pkg} notation

### PR DESCRIPTION
**Pull request by lorenzwalthert**

### CODE REVIEW

**Comment by IndrajeetPatil** [comment]
@lorenzwalthert This has deviated significantly from `main`. Are you still planning to work on this?

**Comment by lorenzwalthert** [comment]
No, kind of waiting for the respective tidyverse style issue to be addressed.

File: R/testing-public-api.R [hunk 1]
```diff
@@ -36,7 +36,7 @@ ls_testable_encodings <- function() {
 
 #' Test the dry argument
 #' @param path A path to pass to the `styler`.
-#' @param styler A function that takes `path`, typically a user exposed styler
+#' @param styler A function that takes `path`, typically a user exposed \{styler\}
```
Comment (krlmlr) [inline R/testing-public-api.R:39]: This looks like a false positive.

**Comment by IndrajeetPatil** [comment]
> No, kind of waiting for the respective tidyverse style issue to be addressed.

Which issue are you referring to?

**Comment by lorenzwalthert** [comment]
https://github.com/tidyverse/style/issues/180. 

But yeah this has so many merge conflicts that we will start from scratch again I guess.

**Pull request closed without merging**